tools/docker/docker-installation.md

```diff
@@ -8,12 +8,39 @@ order: 1
 
 ## Installation
 
-On Fedora 22 and above install the `docker` package:
+On Fedora 30 and above install the `docker-ce` package using the Docker repository:
 
+> for older versions of Fedora see: [docker-installation-fedora-22-29](./docker-installation-fedora-22-29.html)
+
+To install the dnf-plugins-core package (which provides the commands to manage your DNF repositories) and set up the stable repository.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+```
+To add the `docker-ce` repository
+
+```
+$ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
 ```
-$ sudo dnf install docker
+
+To install the docker engine 
+
+```
+$ sudo dnf install docker-ce docker-ce-cli containerd.io
 ```
 
+> to install other version (i.e. nightly versions) see the [Docker documentation](https://docs.docker.com/engine/install/fedora/)
```

Comment:
Could you please enlighten me what `>` in the beginning means?
Also, would you recommend nightly for casual use? If not, please remove this comment. (For explanation please see our [contributing guide](https://developer-portal.github.io/contributing/write-general-writing-hints.html) and [wiki](https://fedoraproject.org/wiki/Websites/Developer).)

Reply:
the `>` at the beginning signifies a quoted section. I will remove as it is not recommended for casual use.